sql/src/main/java/org/apache/druid/sql/guice/SqlBindings.java

```diff
@@ -41,4 +45,12 @@ public static void addOperatorConversion(
   {
     Multibinder.newSetBinder(binder, SqlOperatorConversion.class).addBinding().to(clazz);
   }
+
+  /**
+   * Returns a multiBinder that can modules can use to bind {@link DruidCalciteSchema} to be used by the SqlModule
+   */
+  public static Multibinder<DruidCalciteSchema> calciteSchemaBinder(final Binder binder)
```

Comment:
Maybe do `addSchema` to be consistent with the others.

I recognize this isn't consistent with how DruidBindings works, but you can't win 'em all. (Well, I guess you could win them all, if you changed DruidBindings too.)